libs/application/api/payment/src/lib/payment-callback.controller.ts

```diff
@@ -21,5 +27,18 @@ export class PaymentCallbackController {
       callback.receptionID,
       applicationId,
     )
+
+    const application = await this.applicationService.findOneById(applicationId)
+    if (application) {
+      const oneMonthFromNow = new Date()
+      oneMonthFromNow.setMonth(oneMonthFromNow.getMonth() + 1)
```

Comment:
What if current month is 12, wont this be setting the month to 13?

I would just use `date-fns`

```ts
import addMonths from 'date-fns/addMonths'

const oneMonthFromNow = addMonths(new Date(), 1)
```